### PR TITLE
redis 4.0.5

### PR DESCRIPTION
**Pull request by rakkesh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/redis.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Redis < Formula
   desc "Persistent key-value database, with built-in net interface"
   homepage "https://redis.io/"
-  url "http://download.redis.io/releases/redis-4.0.4.tar.gz"
-  sha256 "35768145335e874b1b810e23494ad3daa6f442c3dc1d7e3784992ba50799c0cd"
+  url "https://github.com/antirez/redis/archive/4.0.5.tar.gz"
```
Comment (ilovezfs) [inline Formula/redis.rb]: This should continue to use the canonical URL

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by rakkesh** [comment]
Changed the download `url` to `github` releases as the earlier version `4.0.4` was failing on sha256 match,

```
==> Downloading http://download.redis.io/releases/redis-4.0.4.tar.gz
Already downloaded: /Users/rakkesh/Library/Caches/Homebrew/redis-4.0.4.tar.gz
Error: SHA256 mismatch
Expected: 35768145335e874b1b810e23494ad3daa6f442c3dc1d7e3784992ba50799c0cd
Actual: 45ca3a8f3576c96721743ea9d589259f30e6dc42d4d5e5b51d842fa9aede8911
```

**Comment by ilovezfs** [comment]
That's a reason to file an issue with them but not a reason to use something other than the canonical URL.

**Review by ilovezfs — APPROVED** [review]

File: Formula/redis.rb (hunk 1 above)
Comment (rakkesh) [reply Formula/redis.rb]: I was under the impression that `Homebrew` prefers `HTTPS` URLs over plain `HTTP`, when available from the upstream project maintainer. Anyway, I have changed it back to the conventional URL.

Comment (ilovezfs) [reply Formula/redis.rb]: Yep, but canonical http over non-canonical https

**Comment by ilovezfs** [comment]
Thanks again @rakkesh!

**Pull request closed without merging**